subsys/bluetooth/services/alexa_gadgets/Kconfig.gadgets

```diff
@@ -15,31 +27,31 @@ config GADGETS_PROFILE_ENABLE
 	help
 	  Enable Alexa Gadgets profile.
 
-if GADGETS_PROFILE_ENABLE
+if BT_GADGETS_PROFILE_ENABLE
```

Comment:
_ENABLE is weird to add to the Kconfig names.

If possible I would remove that as a suffix, I see there are other kconfigs that have this as well.

Reply:
Removing suffix